eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -313,4 +345,80 @@ public function extractFilesFromXml($xml)
 
         return null;
     }
+
+    public function getAllImagesData(): array
```

Comment:
The method should be declared on `\eZ\Publish\Core\FieldType\Image\ImageStorage\Gateway`